perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_defaultDataLimit.java

```diff
@@ -121,7 +97,7 @@ public void checkAttributeSemantics(PerunSessionImpl perunSession, Resource reso
 			if (limitNumber.compareTo(BigDecimal.valueOf(0)) != 0) {
 				throw new WrongReferenceAttributeValueException(attribute, attrDefaultDataQuota, resource, null, resource, null, "Try to set limited limit, but there is still set unlimited Quota.");
 			}
-		} else if ((quotaNumber.compareTo(BigDecimal.valueOf(0)) != 0) && (limitNumber.compareTo(BigDecimal.valueOf(0)) != 0) && defaultDataLimitLetter != null && defaultDataQuotaLetter != null) {
+		} else if ((quotaNumber.compareTo(BigDecimal.valueOf(0)) != 0) && (limitNumber.compareTo(BigDecimal.valueOf(0)) != 0) && defaultDataQuotaLetter != null) {
```

Comment:
Why do you need to check if defaultDataQuotaLetter is null when it came from the same method as defaultDataLimitLetter?